example/spec/helpers/testUtils.js

```diff
@@ -196,18 +196,26 @@ async function redeploy(config, options = {}) {
   );
 }
 
+async function getFileMetadata(file) {
```

Comment:
The changes to this file are just a refactor, they should not alter functionality.